src/server/services/validate-upload.js

```diff
@@ -19,15 +19,15 @@ async function validateAgencyId ({ upload, records, trns }) {
 
   // must be set
   if (!agencyCode) {
-    return new ValidationError('Agency code must be set', { tab: 'cover', row: 1, col: 0 })
+    return new ValidationError('Agency code must be set', { tab: 'cover', row: 1, col: 'A' })
```

Comment:
This was inconsistent: This label assumed it was 0-indexed but everything else assumed A=1 and counted from there